src/time.cc

```diff
@@ -2,109 +2,117 @@
 #include <stdint.h>
 #include <time.h>
 
-#include "nan.h"
+#include "napi.h"
+#include "uv.h"
+#include "math.h"
 
-using namespace node;
-using namespace v8;
+using namespace Napi;
```

Comment:
Is this needed? I think everything is using the namespace explicitly.

Reply:
I fixied it 9b6f5f21fccd6b4a432f9fb0ba4b6750436dc49f